stdlib/public/core/Builtin.swift

```diff
@@ -79,14 +79,35 @@ func _canBeClass<T>(_: T.Type) -> Int8 {
   return Int8(Builtin.canBeClass(T.self))
 }
 
-/// Returns the bits of `x`, interpreted as having type `U`.
+/// Returns the bits of the given instance, interpreted as having the specified
+/// type.
 ///
-/// - Warning: Breaks the guarantees of Swift's type system; use
-///   with extreme care.  There's almost always a better way to do
-///   anything.
+/// Only use this function to convert the instance passed as `x` to a
+/// layout-compatible type when the conversion is not possible through other
+/// means.
 ///
+/// - To convert an integer value from one type to another, use an initializer
```

Comment:
IMO, this is a bit of a jump. I think you need a connective that communicates that you are provided a list of "other means" for common operations when people instinctively reach for unsafeBitCast.

There is no reason not to be more explicit... thoughts?